scripts/lib.sh

```diff
@@ -46,29 +48,67 @@ query-apple-m1() {
 }
 
 get-pyenv-version() {
+    if [[ -n "$SENTRY_PYTHON_VERSION" ]]; then
+        echo "${SENTRY_PYTHON_VERSION}"
+        return 0
+    fi
+
     local PYENV_VERSION
     PYENV_VERSION=3.6.13
     if query-apple-m1; then
-        PYENV_VERSION=3.8.11
+        PYENV_VERSION=3.8.12
```

Comment:
Will this mean anyone currently using M1 (I assume basically no one) will have to rebuild their venv?

Reply:
Yeah. I'm targetting 3.8.12.

Reply:
Hey, hey! Not no one! I'm still here, everyone! :)

wfm